ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/ApplyUniqueValuesWithAlternateSymbols.qml

```diff
@@ -0,0 +1,146 @@
+// [WriteFile Name=ApplyUniqueValuesWithAlternateSymbols, Category=Layers]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.15
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+
+        Map {
+            initBasemapStyle: Enums.BasemapStyleArcGISTopographic
+
+            // Create the feature layer
+            FeatureLayer {
+                id: featureLayer
+
+                // Feature table
+                ServiceFeatureTable {
+                    id: featureTable
+                    url: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0"
+                }
+            }
+            onLoadStatusChanged: {
+
+                if (loadStatus === Enums.LoadStatusLoaded) {
+                    mapView.setViewpoint(initialViewpoint);
+                }
+                // Create the class breaks renderer with alternate symbols and assign it to the feature layer renderer
+                featureLayer.renderer = createUniqueValueRenderer_();
+            }
+        }
+    }
+    ViewpointCenter {
+        id: initialViewpoint
+        center: Point {
+            x: -13631305.660131
+            y: 4546909.846004
```

Comment:
These values were inconsistent with the C++ implementation
```suggestion
            x: -13631205.660131
            y: 4546829.846004
```

Reply:
While we're on the subject, these values are meters and including decimal places with an accuracy to the micrometer may look a little silly to a GIS professional